### PR TITLE
dependencies change diff

### PR DESCRIPTION
**Pull request by dantavori**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-7722

## Description
https://xsoar.docs.pan.run/-/content/-/jobs/30815976/artifacts/artifacts/xsoar/pack_dependencies_diff.json
https://xsoar.docs.pan.run/-/content/-/jobs/30815977/artifacts/artifacts/marketplacev2/pack_dependencies_diff.json
https://xsoar.docs.pan.run/-/content/-/jobs/30815978/artifacts/artifacts/xpanse/pack_dependencies_diff.json

## Must have
- [x] Tests
- [ ] Documentation 

**No mandatory dependencies were added on this pull request.**

### CODE REVIEW

File: Tests/scripts/find_pack_dependencies_changes.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import logging as logger
+from argparse import ArgumentParser
+from pathlib import Path
+
+from demisto_sdk.commands.common.logger import logging_setup
+
+from Tests.scripts.gitlab_client import GitlabClient
+from Tests.scripts.utils.log_util import install_logging
+
+logging_setup(3)
```
Comment (daryakoval) [inline Tests/scripts/find_pack_dependencies_changes.py:11]: What is 3?

**Review by daryakoval — COMMENTED** [review]
`- To consider: comment the diff in the PR, perhaps only if the change is related to core packs.` - I think it is a good idea :)

File: Tests/scripts/gitlab_client.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+from tempfile import mkdtemp
+import zipfile
+from pathlib import Path
+from typing import Any
+
+import requests
+
+
+API_BASE_URL = "https://code.pan.run/api/v4"  # disable-secrets-detection
+PROJECT_ID = "2596"
```
Comment (daryakoval) [inline Tests/scripts/gitlab_client.py:12]: @michal-dagan Have you created a global variable for this?

File: Tests/scripts/get_previous_master_sha.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            shift
+            ;;
+        *)
+            echo "Unknown option: $1"
+            exit 1
+            ;;
+    esac
+done
+
+if [[ $CI_COMMIT_BRANCH = "master" ]]; then
```
Comment (daryakoval) [inline Tests/scripts/get_previous_master_sha.sh:22]: Maybe you mean this? 
```suggestion
if [[ $CI_COMMIT_BRANCH == "master" ]]; then
```

File: Tests/scripts/gitlab_client.py (hunk 2 above)
Comment (michal-dagan) [reply Tests/scripts/gitlab_client.py:12]: yes, please use the predefined variables:

GITLAB_SERVER_URL = get_env_var('CI_SERVER_URL', 'https://code.pan.run')  # disable-secrets-detection
GITLAB_PROJECT_ID = get_env_var('CI_PROJECT_ID', '2596')  # the default is the id of the content project in code.pan.run

GITLAB_CONTENT_PIPELINES_BASE_URL = 'http://code.pan.run/api/v4/projects/2596/pipelines/'  # disable-secrets-detection
# disable-secrets-detection
GITLAB_CONTENT_PIPELINES_BASE_URL = f'{GITLAB_SERVER_URL}/api/v4/projects/{GITLAB_PROJECT_ID}/pipelines/'

**Review by daryakoval — APPROVED** [review]

**Pull request merged**